WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductFromImage/AddProductFromImageViewModel.swift

```diff
@@ -146,10 +151,17 @@ private extension AddProductFromImageViewModel {
         Task { @MainActor in
             do {
                 let texts = try await imageTextScanner.scanText(from: image)
+                guard texts.isNotEmpty else {
+                    throw ScanError.noTextDetected
+                }
+
                 analytics.track(event: .AddProductFromImage.scanCompleted(source: addProductSource, scannedTextCount: texts.count))
                 scannedTexts = texts.map { .init(text: $0, isSelected: true) }
                 [nameViewModel, descriptionViewModel].forEach { $0.reset() }
                 generateProductDetails()
+            } catch ScanError.noTextDetected {
+                shouldShowNoTextDetectedMessage = scannedTexts.isEmpty
+                DDLogError("⛔️ No text detected from image.")
             } catch {
                 analytics.track(event: .AddProductFromImage.scanFailed(source: addProductSource, error: error))
                 DDLogError("⛔️ Error scanning text from image: \(error)")
```

Comment:
Should we also show the message here?

Reply:
Started showing a different error message in https://github.com/woocommerce/woocommerce-ios/pull/10282/commits/f9dc80f70d7df365c78dd6d34a9dca4e2aee83f9